frontend/src/components/Trade/Trade.js

```diff
@@ -0,0 +1,143 @@
+                            {['radio'].map((type) => (
+                                <div key={`stop-${type}`} className="stop-radio">
+                                    <Form.Check
+                                        type={type}
+                                        id={`stop-${type}`}
+                                        label="Stop $"
+                                    />
+                                </div>
+                            ))}
+                            <Form.Group controlId="stopStock">
+                                <Form.Control type="stopStock" placeholder="Enter Stop Price" style={{ border: 'none' }} />
+                            </Form.Group>
+                        </div>
+                    </div>
+                </div>
+                <div className="duration-section">
+                    <h3 className="duration-title" style={{ fontSize: '1.25rem' }}>Duration:</h3>
+                    <Dropdown className="duration-dropdown">
+                        <Dropdown.Toggle variant="success" id="dropdown-basic">
+                            Duration
+                        </Dropdown.Toggle>
+                        <Dropdown.Menu>
+                            <Dropdown.Item href="#/action-1">Good Till Cancelled</Dropdown.Item>
+                            <Dropdown.Item href="#/action-2">Day Order</Dropdown.Item>
+                        </Dropdown.Menu>
+                    </Dropdown>
+                </div>
+                <Button className="previewOrder" variant="primary" onClick={order}>
+                    Preview Order
+                </Button>
+            </div>
+            <div className="account-details-information">
+                <h2 className="account-details">ACCOUNT DETAILS:</h2>
+                <div className="value-section">
+                    <h3 className="value-title">Value (USD):</h3>
+                    <Form>
+                        <input className="value-display" type="text" placeholder="Value" readOnly style={{ border: 'none' }} />
+                    </Form>
+                </div>
+                <div className="buying-power-section">
+                    <h3 className="buying-power">Buying Power:</h3>
+                    <Form>
+                        <input className="buying-power-display" type="text" placeholder="Buying Power" readOnly style={{ border: 'none' }} />
+                    </Form>
+                </div>
+                <div className="cash-section">
+                    <h3 className="cash">Cash: </h3>
+                    <Form>
+                        <input className="cash-display" type="text" placeholder="Cash" readOnly style={{ border: 'none' }} />
+                    </Form>
+                </div>
+                <div className="stock-square">
+                    <small>swuare</small>
+                </div>
+            </div>
+        </div>
+    );
+}
+
+export default Trade;
```

Comment:
Is the all this trading stuff part of another PR?

Reply:
If so, can we just merge this and close the other one

Reply:
yeah we can merge this